install_config/aggregate_logging.adoc

```diff
@@ -1839,8 +1807,8 @@ You can determine which driver Docker is using with the `docker info` command:
 Logging Driver: journald
 ----
 
-To change between `json-file` and `journald` after installation:
- 
+To change to `json-file`:
+
 . Modify either the *_/etc/sysconfig/docker_* or *_/etc/docker/daemon.json_* files.
 +
 For example:
```

Comment:
Are these steps for ensuring changing to `json-file` correct? It is my understanding that the step to restart docker is not correct. Please provide guidance to correct it. 

Can the "Updating Fluentd's Log Source After a Docker Log Driver Update" section be removed entirely? 

Is `USE_JOURNAL` deprecated?

Reply:
The restart of docker looks fine.

I don't know the defaults of what fluentd needs to do on transition. I feel like @richm said that current versions automatically detect what's in use but I'm not sure.

Reply:
USE_JOURNAL is deprecated.  Fluentd assumes both /etc/sysconfig/docker and /etc/docker/daemon.json are mounted into the fluentd pod from the host.  Fluentd will look at these config files to see which log driver docker is using.

Reply:
Fluentd has to be restarted if the docker log driver changes.

Reply:
But - there is a big caveat here - if you are using json file, and switch to journald, and you still have logs in json files - when fluentd is restarted, fluentd will not be able to lookup kubernetes on those json-file logs, and will not be able to do json parsing on the embedded log field on those logs.  If you are going to change the docker log driver, best to ensure all of the json-file logs are drained first.  Similarly with switching from journald to json-file - if there are any container logs still in the journal, once fluentd is restarted, fluentd will not be able to add kubernetes metadata to those logs, nor will be able to parse the embedded json in the MESSAGE field.

Reply:
Thanks @sdodson and @richm! 

I'm removing instructions for switching to/using `journald`, so the caveat *should* be okay. I removed the `USE_JOURNAL` content from the "Updating Fluentd's Log Source After a Docker Log Driver Update" section, but left the steps for now.